openedx_caliper_tracking/processor.py

```diff
@@ -61,7 +61,7 @@ def deliver_caliper_event(caliperized_event, event_type):
             headers={
                 'Authorization': 'Bearer {}'.format(
                     settings.CALIPER_DELIVERY_AUTH_TOKEN),
-                'Content-Type': 'application/json',
+                'Content-Type': 'application/vnd.kafka.json.v2+json',
             },
             json={
```

Comment:
According to requests documentation, `Content-Type` in the header will set to `application/json` but we need `application/vnd.kafka.json.v2+json` please look into this.

REF: [Link](http://docs.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests)

Reply:
good catch @imhassantariq

Reply:
I checked it with `json` and it worked fine. However, I still modified and replaced with data. Just being careful.